### DISCUSSION TITLE
How do you launch your aplication?

### DISCUSSION THREAD

**Discussion by SergioElez** [magidc/nvim-config#3] [Q&A]
Hi! I'm a beginner with Neovim. I have tried your configuration and I don't know how to launch a web application, in my case with my nvim config I work with spring boot and to launch the web application I use Tomcat with the nvim-wildcat plugin but sometimes it gives problems in debug mode, so the question is: how would you launch your web application with this config? Thanks!!

**Comment by magidc** [comment]
Hi, I have no idea how to launch Spring boot apps based on Tomcat. Usually I work on applications that are self-deployable with their own embedded server, it is the most commons way to develop with Spring boot nowadays.
I have never used nvim-wildcat before so I can not offer you help with this issue, sorry.
